### DISCUSSION TITLE
Welcome to Green Software Patterns Discussions!

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dubrie** [comment]
👋 Hello!  I'm Bill and I am the chair of the Patterns group and contribute to the Principles group as well. I've been working in and around green software for about 3 years and am specifically interested in the connection between carbon efficiency and system reliability

**Comment by jawache** [comment]
Hi I'm Asim, I'm the director of the Green Software Foundation, been working in the area of Green Software also for around 3 years, my interest is in reducing the harmful impact from using software.
